src/js/libgeo/GeoOps.js

```diff
@@ -1211,10 +1211,14 @@ geoOps.CircleBy3 = {};
 geoOps.CircleBy3.kind = "C";
 geoOps.CircleBy3.updatePosition = function(el) {
     var a = csgeo.csnames[(el.args[0])].homog;
+    el.startPoint = a;
     var b = csgeo.csnames[(el.args[1])].homog;
+    el.viaPoint = b;
     var c = List.ii;
     var d = List.jj;
     var p = csgeo.csnames[(el.args[2])].homog;
+    el.endPoint = p;
+
```

Comment:
I'd still prefer to have these points not stored for a normal circle. Unless you have a good reason to do otherwise.